File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_14.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// модель значений параметров настройки
public class MB_S_14 {

    /// Название параметра
    @JsonProperty("PARAMNAME")
    @Getter private String paramname;

    /// Значение параметра
    @JsonProperty("PARAMVALUE")
    @Getter private String paramvalue;

    public MB_S_14(String paramname, String paramvalue) {
        this.paramname = paramname;
        this.paramvalue = paramvalue;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_14.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// модель значений параметров настройки
class MB_S_14(
        /// Название параметра
        @SerializedName("PARAMNAME")
        val paramname: String,

        /// Значение параметра
        @SerializedName("PARAMVALUE")
        val paramvalue: String
)